ci-operator/step-registry/ipi/conf/azure/identity/ipi-conf-azure-identity-commands.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+CONFIG_PATCH="${SHARED_DIR}/install-config-identity-patch.yaml"
+
+identity_json_file="${SHARED_DIR}/azure_user_assigned_identity.json"
+
+end_number=$((AZURE_USER_ASSIGNED_IDENTITY_NUMBER - 1))
+if [[ "${ENABLE_AZURE_IDENTITY_DEFAULT_MACHINE}" == "true" ]]; then
+    cat >> "${CONFIG_PATCH}" << EOF
+platform:
+  azure:
+    defaultMachinePlatform:
+      identity:
+        type: UserAssigned
+        userAssignedIdentities:
+EOF
+    for num in $(seq 0 ${end_number}); do
```

Comment:
When there are multiple identifies, how the machines will use them ?

Reply:
All given identity will be attached to machine, and application running on VM can use any of identity to access cloud API ( which I tested when installation running in azure vm with managed identity attached).

Reply:
Ignore the comment, from ci-operator/step-registry/cucushift/installer/check/azure/identity/cucushift-installer-check-azure-identity-commands.sh, I guess I know a bit how it works.